app/views/support_interface/application_forms/application_choices/confirm_reinstate_offer.html.erb

```diff
@@ -0,0 +1,49 @@
+<% content_for :browser_title, title_with_error_prefix('Reinstate offer', @declined_course_choice.errors.any?) %>
+<% content_for :before_content, govuk_back_link_to(support_interface_application_form_path(application_form_id: params[:application_form_id]), 'Back to application') %>
+
+<%= form_with(
+  model: @declined_course_choice,
+  url: support_interface_application_form_reinstate_offer_path(application_form_id: params[:application_form_id], application_choice_id: params[:application_choice_id]),
+  method: :patch
+  )do |f| %>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <%= f.govuk_error_summary %>
+      <h1 class="govuk-heading-l">
+        Are you sure you want to reinstate the offer for this course choice?
+      </h1>
+
+      <h3 class="govuk-heading-s govuk-!-margin-bottom-2">An offer can only be reinstated if:</h3>
+      <ul class="govuk-list govuk-list--bullet">
+        <li>it was declined within 5 working days of the request to reinstate and,</li>
+        <li>the candidate has not accepted any other offers</li>
+      </ul>
+      <p class="govuk-body"> In order to reinstate the offer you must first contact the provider to confirm that they agree to this.</p>
+      <p class="govuk-body"> By reinstating the offer the DBD deadline for this offer, and any other offers will be reset to 10 working days.</p>
+      <p class="govuk-body"> Once the offer has been reinstated, please email the candidate using the macro.</p>
+      <p class="govuk-body"> There are separate macros if the request was made after 5 working days or if the candidate has already accepted another offer.</p>
+    </div>
+  </div>
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <%= f.govuk_text_field(
+              :audit_comment_ticket,
+              label: {
+                text: 'Zendesk ticket URL',
+                size: 'm',
+              },
+              rows: 1,
+              hint: { text: 'e.g. https://becomingateacher.zendesk.com/agent/tickets/12345' },
+            ) %>
+    </div>
+  </div>
+
+  <div class="govuk-form-group">
+    <%= f.govuk_check_boxes_fieldset :accept_guidance, legend: { text: 'Terms of use', size: 'm' } do %>
+      <%= f.govuk_check_box :accept_guidance, true, multiple: false, label: { text: 'I have read the guidance' }, link_errors: true %>
+    <% end %>
+  </div>
```

Comment:
You don’t need this wrapper; it’s included as part of the markup the form builder generates.

```suggestion
  <%= f.govuk_check_boxes_fieldset :accept_guidance, legend: { text: 'Terms of use', size: 'm' } do %>
    <%= f.govuk_check_box :accept_guidance, true, multiple: false, label: { text: 'I have read the guidance' }, link_errors: true %>
  <% end %>
```